cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -614,7 +614,7 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
     String initActions = getString(properties, "initActions");
     boolean runtimeJobManagerEnabled = Boolean.parseBoolean(properties.get(RUNTIME_JOB_MANAGER));
     String autoScalingPolicy = getString(properties, "autoScalingPolicy");
-    int idleTTL = getInt(properties, CLUSTER_IDLE_TTL_MINUTES, skipDelete ? 0 : CLUSTER_IDLE_TTL_MINUTES_DEFAULT);
+    int idleTTL = getInt(properties, CLUSTER_IDLE_TTL_MINUTES, CLUSTER_IDLE_TTL_MINUTES_DEFAULT);
```

Comment:
If I understand correctly, even if `skipDelete` is set to true, the cluster may be automatically deleted after the TTL anyways? Seems like this behavior is rather unintuitive, especially if the `skipDelete` option is used for debugging rather than for batching pipeline runs.

Reply:
Yes. But on the other hand I am worrying if people will start leaving clusters behind and then complain for it. Even for debug they'd better put some TTL even if it's days

Reply:
I confirmed with @bdmogal 30 minutes default is good for this case

Reply:
SGTM, thanks!